### ISSUE TITLE
About Video inpainting results

### ISSUE THREAD

**Issue by Mr-Kyrie** [zibojia/COCOCO#10]
Hi 
I tested a short 10 frame videos. but the inpainting result is poor.
anything wrong about my testing?
input resolution is 432 × 240 and without any promot.
![1728458028708](https://github.com/user-attachments/assets/8acdc35c-f6bc-4ba3-bb12-93da80c4e84e)

**Comment by zibojia** [comment]
We have noticed this problem, this is mainly due to that the model that you used is not the best model. We will released the best model which can remove objects without prompt after this paper is accepted.

**Comment by Mr-Kyrie** [comment]
Do you mean The CoCoCo checkpoints are not the best?

**Comment by zibojia** [comment]
Yes, the model on onedrive is the weak version.

**Comment by Mr-Kyrie** [comment]
OK, Thanks for replying. 
I have another question, is the video length used for inference only applicable to shorter videos, about 10-20 frames?
I have predicted the video using 96-frame video will cause dimension mismatch problems, just like **RuntimeError: The size of tensor a (11088) must match the size of tensor b (10000) at non-singleton dimension 1**

**Comment by zhangp365** [comment]
> OK, Thanks for replying. I have another question, is the video length used for inference only applicable to shorter videos, about 10-20 frames? I have predicted the video using 96-frame video will cause dimension mismatch problems, just like **RuntimeError: The size of tensor a (11088) must match the size of tensor b (10000) at non-singleton dimension 1**

I have the same problem, and I modify the max_len to 20000, and it passes.
![image](https://github.com/user-attachments/assets/f3deea99-7f1e-4385-97d5-b53d5bbb5af7)
I can't find any inpainting effect.
![image](https://github.com/user-attachments/assets/fa8de25a-539d-485f-a4c9-449bd2c07778)
when running, It is often OOM on my two 4090 with 24 vram.

**Comment by mychina75** [comment]
> We have noticed this problem, this is mainly due to that the model that you used is not the best model. We will released the best model which can remove objects without prompt after this paper is accepted.

sorry to say, why could you release the best model only after paper is accpeted? 
Show the better result during paper under review could help the accepetance, maybe.

**Comment by zibojia** [comment]
Yes, we will release the best model after its acceptance.

**Comment by zibojia** [comment]
> > We have noticed this problem, this is mainly due to that the model that you used is not the best model. We will released the best model which can remove objects without prompt after this paper is accepted.
> 
> sorry to say, why could you release the best model only after paper is accpeted? 
> Show the better result during paper under review could help the accepetance, maybe.

Yes, we will release the best model after its acceptance. The model uses same arch but with more pleasing removal effect and won't generate the shutterstock watermark.

**Comment by muratali016** [comment]
@zibojia  @Mr-Kyrie @zhangp365 hi guys where can I try this model with an interface like yours, on huggingface space? really appreciated any response

**Comment by zhangp365** [comment]
> @zibojia @Mr-Kyrie @zhangp365 hi guys where can I try this model with an interface like yours, on huggingface space? really appreciated any response

not hugging face space, just local machine

**Comment by muratali016** [comment]
@zhangp365 can you please share the local code

**Comment by muratali016** [comment]
@zhangp365 hi again :d
